data/json/npcs/godco/NPC_Corrie_Kaja_Dosia.json

```diff
@@ -0,0 +1,229 @@
+          { "u_lose_var": "unclaimed_machine_reward", "type": "mission", "context": "reward" }
+        ],
+        "topic": "TALK_DONE"
+      }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Corrie_Result_rewardRation",
+    "dynamic_line": "Yeah, yeah they have.  I've noticed a change in attitude towards us, the new guys.  Positive ones.  Things might be looking up, a little bit.",
+    "responses": [
+      {
+        "text": "How about some food as payment?",
+        "topic": "TALK_GODCO_Corrie_Result_rewardIcon1",
+        "effect": [
+          { "npc_add_var": "godco_corrie_machine", "type": "mission", "context": "completed", "value": "yes" },
+          { "u_add_var": "godco_corrie_machine", "type": "mission", "context": "completed", "value": "yes" },
+          { "u_lose_var": "unclaimed_machine_reward", "type": "mission", "context": "reward" }
+        ]
+      },
+      {
+        "text": "Maybe you can teach me something as payment?",
+        "topic": "TALK_TRAIN",
+        "condition": "npc_train_skills",
+        "effect": [
+          { "npc_add_var": "godco_corrie_machine", "type": "mission", "context": "completed", "value": "yes" },
+          { "u_add_var": "godco_corrie_machine", "type": "mission", "context": "completed", "value": "yes" },
+          { "u_lose_var": "unclaimed_machine_reward", "type": "mission", "context": "reward" }
+        ]
+      },
+      {
+        "text": "Glad to help.  What were you saying before?",
+        "topic": "TALK_NONE",
+        "effect": [ { "u_add_var": "godco_corrie_machine", "type": "mission", "context": "completed", "value": "yes" } ]
+      },
+      {
+        "text": "Glad to help.  See you around, Corrie.",
+        "topic": "TALK_DONE",
+        "effect": [ { "u_add_var": "godco_corrie_machine", "type": "mission", "context": "completed", "value": "yes" } ]
+      }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Corrie_Result_rewardIcon1",
+    "dynamic_line": "Oh, I completely forget.  Here you are.",
+    "speaker_effect": { "effect": [ { "u_spawn_item": "jerky", "count": 5 } ] },
+    "responses": [
+      { "text": "What were you saying before?", "topic": "TALK_NONE" },
+      { "text": "I'd better get going.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Corrie_Mission",
+    "dynamic_line": "Basic repairs and upkeep, mostly.  The generator in the basement is on its last legs, so I've been thinking about building a solar array for this place.  Helena's against it, but I want to change her mind.  All I need are a few spare parts.",
+    "responses": [
+      { "text": "I can help you get the items you need.", "topic": "TALK_MISSION_LIST" },
+      { "text": "What were you saying before?", "topic": "TALK_NONE" },
+      { "text": "I'd better get going.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_GODCO_Corrie_Recruit",
```

Comment:
So if I followed that questline correctly, it looks like you both work together to create a solar array for the compound, Corrie starts getting more respect from the community as a response to the solar array, and after that you can recruit Corrie to come with you just when they are starting to be accepted by the community?

Reply:
That's the way it seems to go. 

I guess Corrie decides to come with you because you helped selflessly helped her. If I remember correctly, she will only come with you if you tell Helena to give her extra food, forfeiting your own reward to her benefit. 
The community, on the other hand, only started to tolerate her after she overhauled their power supply, in stark contrast to what they preach. She could see that disparity, and choose to go with the Good Samaritan who holds up their end of things; the player.